server/core/src/storage.ts

```diff
@@ -196,7 +196,7 @@ class TServerStorage implements ServerStorage {
       return result[0]
     }
     if (result.length === 0) {
-      return [{}, false]
```

Comment:
Why we need this change here?

Reply:
For ApplyIf. It's expected to return boolean, but in case with non-complete request it returns array, which we check as boolean and it's always true (cause it's an array) even though should've been false

Reply:
ok